### PR TITLE
Fixes #445 Added step for installing wget

### LINKED ISSUE #445: Add step for installing wget during HZ installation
Not all Linux distros come with wget installed. Have user verify wget installation before running the commands to install Hazelcast.

Reference doc page: https://docs.hazelcast.com/hazelcast/5.1/getting-started/install-hazelcast

### PR DESCRIPTION
**Pull request by angelasimms**
Updated install-hazelcast.adoc. Linux tab updated to include wget check and install.

### CODE REVIEW

File: docs/modules/getting-started/pages/install-hazelcast.adoc [hunk 1]
```diff
@@ -34,40 +34,53 @@ Linux::
...
+```bash
+wget -v
+```
++
+If you do not see a version number, download and install link:https://www.gnu.org/software/wget/[Wget^].
++
+
+. Use either of these package managers to install Hazelcast, depending on your Linux distribution:
 ifdef::snapshot[]
++
```
Comment (JakeSCahill) [inline docs/modules/getting-started/pages/install-hazelcast.adoc:48]: I think we can remove this + and put it before the first conditional.

File: docs/modules/getting-started/pages/install-hazelcast.adoc [hunk 2]
```diff
@@ -34,40 +34,53 @@ Linux::
 +
 --
 
-To install on Linux, you can use either of these package managers, depending on your Linux distribution:
+. Check that you have the Wget utility installed. Hazelcast uses this utility in its package managers.
```
[the remark this answers is not in the document]
Comment (JakeSCahill) [reply docs/modules/getting-started/pages/install-hazelcast.adoc:37]: GNU Wget is the name used in the official doc you link to. Hazelcast doesn't use it, it's just a tool for downloading resources on the Internet.

File: docs/modules/getting-started/pages/install-hazelcast.adoc (hunk 1 above)
Comment (angelasimms) [reply docs/modules/getting-started/pages/install-hazelcast.adoc:47]: Updated as requested.

File: docs/modules/getting-started/pages/install-hazelcast.adoc (hunk 2 above)
[the remark this answers is not in the document]
Comment (angelasimms) [reply docs/modules/getting-started/pages/install-hazelcast.adoc]: Updated as requested.

File: docs/modules/getting-started/pages/install-hazelcast.adoc [hunk 3]
```diff
@@ -34,40 +34,53 @@ Linux::
...
 [source,shell]
 ----
 wget https://repository.hazelcast.com/rpm/snapshot/hazelcast-rpm.repo -O hazelcast-snapshot-rpm.repo
 sudo mv hazelcast-snapshot-rpm.repo /etc/yum.repos.d/
 sudo yum install hazelcast
 ----
++
 endif::[]
 ifndef::snapshot[]
++
```
[the remark this answers is not in the document]
Comment (angelasimms) [reply docs/modules/getting-started/pages/install-hazelcast.adoc]: Updated as requested.

**Review by JakeSCahill — APPROVED** [review]

**Pull request merged**